features/poller/monitors/core/src/main/java/org/opennms/netmgt/poller/monitors/SnmpMonitor.java

```diff
@@ -107,6 +111,30 @@ public PollStatus poll(MonitoredService svc, Map<String, Object> parameters) {
         String reasonTemplate = ParameterMap.getKeyedString(parameters, "reason-template", DEFAULT_REASON_TEMPLATE);
         String hexstr = ParameterMap.getKeyedString(parameters, "hex", "false");
 
+        String location = ParameterMap.getKeyedString(parameters, "location", LocationUtils.DEFAULT_LOCATION_NAME);
+
+        if (!LocationUtils.DEFAULT_LOCATION_NAME.equals(location) && agentConfig.getAddress().isLoopbackAddress()) {
```

Comment:
Another way to access scv by overriding  `@Override
    public Map<String, Object> getRuntimeAttributes(MonitoredService svc, Map<String, Object> parameters)`. and get read community there and send it as a parameter.

Reply:
As @amay94 pointed, we can't use `getRuntimeAttributes` for this as the scv needs to be retrieved only on Minion and `getRuntimeAttributes` only runs on OpenNMS